src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -197,16 +230,20 @@ export const QueryAssistSummary: React.FC<QueryAssistSummaryProps> = (props) =>
       borderRadius="none"
     >
       <EuiSplitPanel.Inner
-        style={{ background: 'linear-gradient(to left, #EDF7FF, #F9F4FF)', padding: '6px' }}
+        className={
+          isDarkMode
+            ? 'queryAssist queryAssist__summary_banner_dark'
+            : 'queryAssist queryAssist__summary_banner'
```

Comment:
not sure but is there a way to apply correct style based on whether dark mode is enabled inside css instead of conditionally set the class names in javascript?

Reply:
Move the logic to scss